intro-Linux/ch_beyond_the_basics.tex

```diff
@@ -485,55 +498,56 @@ \subsection{PBS pragmas}
 how much memory will it use, how long will it run? These things can be specified
 inside a script with what we call PBS pragmas.
 
+This option tells PBS to use 1 node and 1 core:
+
 \begin{code}{bash}
   #PBS -l nodes=1:ppn=1   # single-core
 \end{code}
 
 For parallel software, you can request multiple cores (OpenMP) and/or multiple
-nodes (MPI). '''Only use this when the software you use is capable of working in
-parallel.'''
+nodes (MPI). \strong{Only use this when the software you use is capable of working in
+parallel.}. Here is an example:
 
 \begin{code}{bash}
   #PBS -l nodes=1:ppn=16  # single-node, multi-core
   #PBS -l nodes=5:ppn=16  # multi-node
 \end{code}
 
-This option tells PBS to use 1 node and 1 core.
+We submit it on the long queue:
 
 \begin{code}{bash}
   #PBS -q long
 \end{code}
 
-We submit it on the long queue.
+We request a total running time of 2 days (48 hours).
 
 \begin{code}{bash}
   #PBS -l walltime=48:00:00
 \end{code}
 
-We request a total running time of 2 days (48 hours).
+We tell PBS the name of our job:
 
 \begin{code}{bash}
   #PBS -N FreeSurfer_per_subject-time-longitudinal
 \end{code}
 
-We tell PBS the name of our job.
+This specifies mail options:
 
 \begin{code}{bash}
   #PBS -m abe
 \end{code}
 
-This specifies mail options.
 \begin{enumerate}
-\item \strong{a} means mail is sent when the job is aborted by the batch system.
-\item \strong{b} means mail is sent when the job begins.
-\item \strong{e} means mail is sent when the job ends.
+\item \verb|a| means mail is sent when the job is aborted by the batch system.
```

Comment:
remove the `by the batch system` (readers may start wondering what a `batch system` is :))